templates/commands/goldentest/test_funcs.go

```diff
@@ -252,3 +253,24 @@ func renameGitDirsAndFiles(dir string) error {
 	}
 	return nil
 }
+
+func parseTemplateLocations(dir string) ([]string, error) {
+	var out []string
+	err := filepath.WalkDir(dir, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.IsDir() {
+			if _, err := os.Stat(filepath.Join(path, "spec.yaml")); err == nil {
+				out = append(out, path)
+			}
```

Comment:
Let's also exclude golden-test directories from being crawled. Some templates, like gcp-org-terraform template, have *output files* named spec.yaml. Therefore, these spec.yaml files will appear in golden test testdata directories.

So we can do something like `if d is a golden test testdata directory, then return filepath.SkipDir`.